prototree/node.py

```diff
@@ -261,7 +261,7 @@ def __init__(
         super().__init__(index, parent=parent)
 
         self.num_classes = num_classes
-        self.dist_params = nn.Parameter(torch.zeros(num_classes), requires_grad=False)
+        self.dist_params: nn.Parameter = nn.Parameter(torch.zeros(num_classes), requires_grad=False)

```

Comment:
I usually leave out types that can be inferred. Any reason you want to annotate explicitly here?

Reply:
Thanks, I'll see if I can make a (more) minimal reproducible example and submit that.